payjoin/src/send/mod.rs

```diff
@@ -561,6 +542,11 @@ impl ContextV1 {
                 }
                 // theirs (receiver)
                 None | Some(_) => {
+                    let original = self
+                        .original_psbt
+                        .input_pairs()
+                        .next()
+                        .expect("original PSBT should have an input");
```

Comment:
Can we handle this error?

Reply:
It would mean the original PSBT had no inputs, which seems improbable at this stage since the receiver would've accepted it and responded with a payjoin proposal. I could add a proper error type for it though.

Reply:
Added proper handling for this error in 68fc635f57bd